### PR TITLE
Calypso signup: Theme selection copy optimization test

### PR DESCRIPTION
**Pull request by fditrapani**
In March, we saw a drop in conversions for our theme selection step at signup. Updating the copy in our theme selection step to reflect choices from the previous step will increase conversions on that step by a minimum of 2%.

## Variations
Copy will be updated to reflect the user's choice from the previous step and also to let them know they can pick from hundreds of other themes at a later point. 

`original`:
screen shot 2017-04-24 at 10 06 26 am

`modified`:
screen shot 2017-04-24 at 10 54 40 am
screen shot 2017-04-24 at 10 54 46 am
screen shot 2017-04-24 at 10 54 52 am

## Test parameters
Test name: `signupThemeStepCopyChanges`
Author: @fditrapani
Domain: Signup
Start: 2017-04-26
End: 2027-05-03
Baseline Conversion Rate: 80.7%
Minimum Detectable Effect: 2%
Statistical Power: 95%
Significance Level: 1%
Weekly Traffic: 244,393
Traffic Allocation: `original`: 50% – `modified`: 50%
Minimum Sample Size: <a href="http://www.evanmiller.org/ab-testing/sample-size.html#!80.7;95;1;1;0">55,910</a>

@markryall @travisw can you please review?

### CODE REVIEW

**Comment by travisw** [comment]
It seems to work as expected.

When clicking the option "A grid of my latest posts" the copy on the next page is "Here are our most popular portfolio designs."
Those two things (grid of posts and portfolio) feel a little disconnected to me. A grid of posts is a fairly generic design. But a portfolio sounds like a very specific type of website that an artist might create to showcase their work.

File: client/signup/steps/theme-selection/index.jsx [hunk 1]
```diff
@@ -58,8 +59,7 @@ class ThemeSelectionStep extends Component {
 			<SignupThemesList
 				surveyQuestion={ this.props.chosenSurveyVertical }
 				designType={ this.props.designType || this.props.signupDependencies.designType }
-				handleScreenshotClick={ this.pickTheme }
-			/>
```
Comment (travisw) [inline client/signup/steps/theme-selection/index.jsx]: I've seen this closing part of the tag written on a newline as well as at the end of the previous line in Calypso. Is there any mention in the coding standards for which is preferred? I haven't seen anything.

**Comment by travisw** [comment]
One other comment. For future PRs, it might be helpful to have more step by step testing instructions to make sure everything you need tested is looked at.

File: client/signup/steps/theme-selection/index.jsx (hunk 1 above)
Comment (markryall) [inline client/signup/steps/theme-selection/index.jsx]: i'd generally go for a new line for symmetry but eslint doesn't express an opinion on the matter.

**Comment by fditrapani** [comment]
> One other comment. For future PRs, it might be helpful to have more step by step testing instructions to make sure everything you need tested is looked at.

Great suggestion @travisw I'll make sure to include them in future PRs

**Comment by fditrapani** [comment]
> When clicking the option "A grid of my latest posts" the copy on the next page is "Here are our most popular portfolio designs."
Those two things (grid of posts and portfolio) feel a little disconnected to me. A grid of posts is a fairly generic design. But a portfolio sounds like a very specific type of website that an artist might create to showcase their work.

@travisw you actually found a bug there. This is where instructions would have helped :P. This test should only be shown to new users, you were going through the create a site flow for existing users rather than the create account + site for new users. I put in a fix thanks to @markryall.

**Comment by markryall** [comment]
👍 looks good

**Pull request merged**